amqp-postgres/amqp_postgres/postgres_publisher.py

```diff
@@ -273,7 +276,8 @@ def _insert_logs(self, cursor, logs):
     def on_db_connection_error(self):
         logger.critical('Database down - cannot continue')
         self._amqp_connection.close()
-        raise RuntimeError('Database down')
+        self.error_exit = RuntimeError('Database down')
```

Comment:
Do we really want to mask the original error? Can we pass through the specific DB error?